org.jacoco.core/src/org/jacoco/core/analysis/Analyzer.java

```diff
@@ -124,9 +123,8 @@ public void analyzeClass(final ClassReader reader) {
 	public void analyzeClass(final byte[] buffer, final String location)
 			throws IOException {
 		try {
-			analyzeClass(
-					new ClassReader(Java9Support.downgradeIfRequired(buffer)));
-		} catch (final RuntimeException cause) {
+			analyzeClass(new ClassReader(buffer));
+		} catch (final Exception cause) {
```

Comment:
Note that here I changed from RuntimeException to Exception

Reply:
Wondering why? how this is related to ASM 6? I'm asking because we had some improvements and quite some nasty regressions/missed cases of exception handling exactly here.

Reply:
Initially, I got some tests with errors (the test were expecting a exception with a actual message. but null were given). 

Later I noticed that the method `Instrumenter. instrument(InputStream...)` need to be wrapped with a try-catch block (noticed by looking the previous state in the git history).

But I didn't know if it was related

Reply:
@Godin I double checked my changes.  Looks that this can be reverted.